doc/release/platforms/udp.rst

```diff
@@ -0,0 +1,161 @@
+.. _using-udp:
+
+==============================
+Using the Portable UDP Conduit
```

Comment:
Is it possible to drop the title only in the links to it, but not here? I like having the full title at the top of the page.